pkg/obfuscate/sql_tokenizer.go

```diff
@@ -205,7 +205,7 @@ func (tkn *SQLTokenizer) Scan() (TokenKind, []byte) {
 	if tkn.lastChar == 0 {
```

Comment:
Why are the changes in this file present?

Reply:
Extracted those changes into https://github.com/DataDog/datadog-agent/pull/10129